client/auth.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Authentication Form</title>
+</head>
+
+<body>
+    <div class="container" id="container">
+        <div class="form-container log-in-container">
+            <form action="#">
+                <h1>Login</h1>
+                <input id="loginUsername" type="username" placeholder="Username" />
+                <input id="loginPassword" type="password" placeholder="Password" />
+                <button onclick="return logIn()">Log In</button>
+            </form>
+        </div>
+        <div class="overlay-container">
+            <div class="overlay">
+                <div class="overlay-panel overlay-right">
+                    <form id="signUpForm">
+                        <h1 id="headerSignup">Create Account</h1>
+                        <input id="signupUsername" type="username" placeholder="Username" value="" />
+                        <input id="signupEmail" type="email" placeholder="Email" value="" />
+                        <input id="signupPassword" type="password" placeholder="Password" value="" />
+                        <button id="signUpButton" onclick="return signUp()">Signup</button>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+    <h3 id="ActionStatus"></h3>
+</body>
+<script src="scripts/auth.js"></script>
+<script>
+    // Zoomutex server url
+    const zoomMutexJoin = "http://127.0.0.1:7000/join"
```

Comment:
We'll have to change this for deploying to heroku, right?

Reply:
Yep, i'll add that in.